setup_go.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -ex
+
+# Install gimme to get go1.15.11
+curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+chmod +x /bin/gimme
+eval $(gimme 1.15.11)
+
+# Install gcc 4.9 (needed to compile go, as it introduces -fdiagnostics-color which is required
+# in the post-compilation steps of go)
+# We don't add it to the PATH so that it's not used in the Agent / rtloader builds
+apt-get update && apt-get install -y gcc-4.9-backport
+
+git clone --branch go$GO_VERSION --depth 1 https://go.googlesource.com/go goroot && cd goroot
+cd src 
```

Comment:
I'm a bit curious why `cd goroot` is inlined above but we cd again here on a separate line. Probably should join those two together.

Reply:
Yeah, it comes from a previous version of this script where the clone + checkout were done in 2 steps.